### ISSUE TITLE
implement HmIP-BSL

### ISSUE THREAD

**Issue by MarkusJankowski** [coreGreenberet/homematicip-rest-api#165]
Hello, 

please implement HMIP-BSL. 
Here is my anonymized config: https://pastebin.com/vwc97DdM

Thanks a lot
and BR
Markus

**Comment by coreGreenberet** [comment]
could you please set the top light to red and the bottom light to green and repost the config?
I would like to know which of the channels is the bottom/top one.

**Comment by MarkusJankowski** [comment]
Lights are on:
https://pastebin.com/grUuzbvs

**Comment by MarkusJankowski** [comment]
Both lights are on with 100%.

**Comment by coreGreenberet** [comment]
thanks =)

**Comment by MarkusJankowski** [comment]
I think it's working.
device.set_rgb_dim_level(device.topLightChannelIndex, RGBColorState.RED, 0.8)
--> turns on the top light with red colour and dimmer level of 80% as expected. I verified this in the APP too.

device.bottomLightChannelIndex, RGBColorState.BLUE, 0.5, 10, 20
--> turns on the buttom light with blue colour and a minimum dimmer level (3%). The ramp-up to a dimmer-level takes 20 secounds and afterwards the light is on for another 10 secounds.   I verified this in the APP too.

The normal switching (turn_on, turn_off) is also working.
